docs/Building.md

```diff
@@ -21,6 +139,7 @@ with `-DCMAKE_TOOLCHAIN_FILE="<path-to-qt>/lib/cmake/Qt6/qt.toolchain.cmake"`
 A minimal example command is:
 
 ```bash
+git submodule update --init --recursive
```

Comment:
I do not like mixing git commands with build commands. I would rather add a separate block above where you remind people that submodules may need to be updated.